actions/duplicate_action.go

```diff
@@ -1,12 +1,14 @@
+//nolint:depguard
```

Comment:
I'm actually okay if you want to tone down `golangci-lint`'s settings a loooooooooot instead of having to riddle the source files with `nolint` directives.

Reply:
How do you feel about dynamic errors? There's a linter that doesn't allow it but I don't see a reason for most, if any, of the errors to be defined statically, that aren't already.

Reply:
I'm pretty ambivalent about it, tbh. Sometimes it makes sense, and sometimes it's just annoying to deal with. I'd say axe that linter.